packages/modular-site/src/App.tsx

```diff
@@ -20,6 +20,7 @@ import './App.css';
 const worker = perspective.shared_worker();
 
 async function getTable(): Promise<Table> {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
```

Comment:
Not sure we should even *have* modular-site tbh :)

Reply:
Shall I leave it like this for now as we're removing it?

Reply:
👍